extensions/amp-viewer-assistance/0.1/amp-viewer-assistance.js

```diff
@@ -173,19 +176,49 @@ export class AmpViewerAssistance {
       return false;
     }
     const update = args['update'];
+    const error = args['error'];
     const actionStatus = update && update['actionStatus'];
-    if (!update || !actionStatus) {
+
+    if (error) {
+      return true;
+    } else if (update) {
+      if (!actionStatus || !ACTION_STATUS_WHITELIST.includes(actionStatus)) {
+        user().error(TAG, '"updateActionState" action "update" parameter' +
+            ' must contain a valid "actionStatus" field.');
+        return false;
+      }
+      return true;
+    } else {
```

Comment:
Optional: `user().warn(...)` if both `error` and `update` are passed.

Reply:
I added an error to stop the call all-together instead of just warning the user. I'd rather not have the ambiguity of which param will be used.